runtime/centrifuge/src/migrations.rs

```diff
@@ -9,36 +9,34 @@
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
-use cfg_primitives::Balance;
-use cfg_types::{
-	tokens::{
-		lp_eth_usdc_metadata, CrossChainTransferability, CurrencyId, CustomMetadata,
-		ETHEREUM_MAINNET_CHAIN_ID, ETHEREUM_USDC, LP_ETH_USDC_CURRENCY_ID,
-	},
-	xcm::XcmMetadata,
-};
-use frame_support::{
-	traits::{Len, OnRuntimeUpgrade},
-	weights::Weight,
-};
-use sp_std::{vec, vec::Vec};
-use xcm::{v3::prelude::*, VersionedMultiLocation};
-
-use crate::{LiquidityPoolsPalletIndex, RocksDbWeight, Runtime};
+use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
 
 pub type UpgradeCentrifuge1020 = (
-	asset_registry::CrossChainTransferabilityMigration,
```

Comment:
Note: Not necessary due to hardcoded metadata from `runtime_common::migrations::asset_registry_xcmv3::Migration`